From d81417b4bab59882a2615971f9bc8e3bba989925 Mon Sep 17 00:00:00 2001
From: Mike Aizatsky <maizatskyi@cloudflare.com>
Date: Tue, 6 Feb 2024 12:55:07 -0800
Subject: increase visibility of virtual method

Methods were marked as protected, which conflicts with
v8-platform-wrapper.h implementation.

diff --git a/include/v8-platform.h b/include/v8-platform.h
index 5110f432077dce9770319373aec4988be1eaff59..6fb67c8c77b0d6adcc6b294230d4488a0fef9bc5 100644
--- a/include/v8-platform.h
+++ b/include/v8-platform.h
@@ -1308,7 +1308,7 @@ class Platform {
     return &default_observer;
   }
 
- protected:
+ public:
   /**
    * Default implementation of current wall-clock time in milliseconds
    * since epoch. Useful for implementing |CurrentClockTimeMillis| if
